### DISCUSSION TITLE
Control over up axis in VR mode?

### DISCUSSION THREAD

**Discussion by paulmelis** [NVlabs/instant-ngp#1216] [Q&A]
Is there a way to specify (or adjust) the up axis of the scene in VR? The two-hand grab rotates around up, which is great, but I have a few scenes where the floors are not exactly the way they should be :-)

**Comment by Tom94** [comment]
Hi, the up-vector is stored with the dataset -- you can change it in `transforms.json` (or, if it's not present, you can add `"up": [x, y, z],` to it).

I'll add GUI controls for the up vector tomorrow, but those will be reset upon restart, because instant-ngp currently doesn't provide a way to update the dataset (by design and unlikely to change). I'll make it export correctly into `.ingp` files, though.

**Reply by Tom94** [reply]
GUI controls for world orientation are added now.

![image](https://user-images.githubusercontent.com/4923655/216962696-c2363c68-58a3-4383-95fe-32977155f253.png)
